### PR TITLE
Feature/relax image

### PR DESCRIPTION
**Pull request by PaulJohannFlade**

### CODE REVIEW

File: components/RandomImage.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    const randomIndex = Math.floor(Math.random() * images.length);
+    setRandomImage(images[randomIndex]);
+  }, []);
+
+  return (
+    <CenteredContainer>
+      {randomImage && (
+        <Image
+          src={randomImage.src}
+          alt="Random"
```
Comment (SabinePfanner) [inline components/RandomImage.js:39]: If you want to set an ```alt text``` I would recommend to use a more descriptive wording.

File: components/RandomImage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          width={randomImage.width}
+          height={randomImage.height}
+          priority
+          style={{ width: "50%", height: "auto" }}
```
Comment (SabinePfanner) [inline components/RandomImage.js:43]: These are already the default settings, not sure if you need to define them again.

**Review by SabinePfanner — APPROVED** [review]
Looks good to me, great work and nice turtle pictures ;) Just 2 small suggestions - up to you if you want to change these.

File: components/RandomImage.js (hunk 1 above)
Comment (PaulJohannFlade) [reply components/RandomImage.js]: Thanks for this hint. I have written a little description in the alt-tag.

File: components/RandomImage.js (hunk 2 above)
Comment (PaulJohannFlade) [reply components/RandomImage.js:43]: Thanks for your hint. I think this is fine so for the first.

**Review by PaulJohannFlade — COMMENTED** [review]
add a little description in alt-tag

**Pull request merged**